index.md

```diff
@@ -15,6 +15,17 @@ tensors. A particular focus of Zarr is to provide support for storage
 using distributed systems like cloud object stores, and to enable
 efficient I/O for parallel computing applications.
 
+![arabidopsis](/images/arabidopsis.png "arabidopsis")
+<center>Image by <a href="https://www.openmicroscopy.org/2020/11/04/zarr-data.html">OME</a></center>
+
+
+Have beautiful data in Zarr? Show us on Twitter and mention [@zarr_dev](https://twitter.com/zarr_dev) to get featured!
+
+>Recent Updates
+
+- [Sanket Verma](https://twitter.com/msankeys963/) has officially joined Zarr as the new Community Manager! 🎉 Read the blog post [here](https://zarr.dev/blog/welcoming-community-manager/).
```

Comment:
For the moment, hand coding these is fine, but eventually we might want something more error proof. Perhaps we can consume the RSS feed from `/blog` directly.

Reply:
Also, I'd probably be inclined to having them at the bottom of the page, but maybe that's better done when we have more subpages since it's quite long at the moment.

Reply:
Noted. Will take care of this in the upcoming weeks!